heart/heartbeat.py

```diff
@@ -98,13 +108,7 @@ def replace_char_all(s, f, t):
 	s = s.replace(chr(0xFC) + chr(0x80) + chr(0x80) + chr(0x80) + chr(0x80 | ((v>>6)&3)) + chr(0x80 | (v&63)), t)
 	# TODO: handle the 6-bit and 8-bit variants and whatnot
 	return s
-
-def sanestr(s):
```

Comment:
There's no real need for this to be moved to HTTPClient, but whatever, no harm in it being there.